poketechnika/Camera.cpp

```diff
@@ -10,9 +10,9 @@ void Camera::update()
 
 bool Camera::isWithinCamera(sf::Vector2i pos)
 {
-	if (pos.x < position.x || pos.y < position.y || pos.x > position.x + size.x + 63 || pos.y > position.y + size.y + 63)
-		return false;
-	return true;
+	return !(
+		pos.x < position.x || pos.y < position.y || 
+		pos.x > position.x + size.x + 63 || pos.y > position.y + size.y + 63);
```

Comment:
to jest dodane tylko zw względu na czytelność

Reply:
szczerze dla mnie czytelniejsza jest pierwsza wersja, ale spoko